guides/source/configuring.md

```diff
@@ -1520,6 +1521,17 @@ The default value depends on the `config.load_defaults` target version:
 | (original)            | `true`               |
 | 7.0                   | `false`              |
 
+#### `config.action_view.prepend_content_exfiltration_prevention`
+
+Determines whether or not the `form_tag` and `button_to` helpers will produce HTML tags prepended with additional matter designed to prevent exfiltration of form data. The default value is `false`.
+
+The default value depends on the `config.load_defaults` target version:
```

Comment:
I don't think this is true, so no need for the table.

Reply:
I'm not sure I understand what you mean. I thought the goal was to leave this feature disabled by default for 7.0 and enable it by default in 7.1. How should the table look to reflect that (I don't really understand what "original" means).